roles/edpm_libvirt/templates/qemu.conf.j2

```diff
@@ -8,3 +8,6 @@ nbd_tls = 0
 # port usage conflicts. See https://review.openstack.org/#/c/561784
 migration_port_min = 61152
 migration_port_max = 61215
+{% if edpm_nova_libvirt_qemu_group is defined%}
+group = "{{ edpm_nova_libvirt_qemu_group }}"
```

Comment:
This group change is specific to DPDK, so setting only for DPDK ansible variable is defined. otherwise no impact on existing config.
Need to configure group value 'hugetlbfs'. otherwise we can't set vhostuser DPDK port.

Reply:
i think, group default value is kvm but not used previously and may create issue in other features. 
we can add new  PR later if any default value required.

Reply:
@SeanMooney please check if this is the way we are supposed to configure things like that for libvirt, or should we instead provide direct config snippets on osdpd level